server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -167,7 +183,14 @@ class LivyServer extends Logging {
             val batchServlet = new BatchSessionServlet(batchSessionManager, sessionStore, livyConf)
             mount(context, batchServlet, "/batches/*")
 
-            context.mountMetricsAdminServlet("/")
+            if (livyConf.getBoolean(UI_ENABLED)) {
+              val uiServlet = new UIServlet(livyConf)
+              mount(context, uiServlet, "/ui/*")
+              mount(context, staticResourceServlet, "/static/*")
+              mount(context, uiRedirectServlet, "/*")
```

Comment:
It does not, I tried just using `"/"` but the `mount` function experts a wildcard in that String